src/components/modules/VehicleOverview/index.jsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
+import PT from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+import vehicleOverviewData from 'services/data/vehicleOverview.js';
 
 import PageHeaderBarContainer from 'common/PageHeaderBarContainer';
 import VehicleTypeList from './VehicleTypeList';
 
-const VehicleOverview = () => (
-  <PageHeaderBarContainer
-    to="/"
-    title="Voertuigen"
-    subTitle=""
-  >
-    <VehicleTypeList/>
-  </PageHeaderBarContainer>
-);
+class VehicleOverview extends React.Component {
```

Comment:
I don't see the reason you would use a class component here instead of a function component